lib/main_publiccloud.pm

```diff
@@ -148,7 +141,7 @@ The rest of the scheduling is divided into two separate subroutines C<load_maint
 =cut
 
 sub load_publiccloud_tests {
-    if (check_var('PUBLIC_CLOUD_TOOLS_REPO', 1)) {
+    if (check_var('PUBLIC_CLOUD_PREPARE_TOOLS', 1)) {
```

Comment:
Is this correct? Looking at https://openqa.suse.de/tests/7241026#settings and the [PC Tools image job group](https://openqa.suse.de/admin/job_templates/276) this looks wrong to me?

Reply:
if you look closely , you will realize that with current state *any* change in `main_publiccloud`  is correct for https://openqa.suse.de/admin/job_templates/276 because of `YAML_SCHEDULE` variable :) so this Job Group certainly require some MR :) but again due to `YAML_SCHEDULE` variable we are safe to do it independently later

Reply:
Ahhh, I see. And you also solved another bit of headscratching. I still get confused by the mixed usage of YAMLSCHEDULE and `main_$whatever.pm` that we have.